keras/callbacks/callbacks.py

```diff
@@ -655,9 +655,9 @@ class ModelCheckpoint(Callback):
             If `save_best_only=True`, the decision
             to overwrite the current save file is made
             based on either the maximization or the
-            minimization of the monitored quantity. For `val_acc`,
-            this should be `max`, for `val_loss` this should
-            be `min`, etc. In `auto` mode, the direction is
+            minimization of the monitored quantity. For `__monitor__=val_acc`,
+            it should be `__mode__=max`, for `__monitor__=val_loss` this should
+            be `__mode__=min`, etc. In `auto` mode, the direction is
```

Comment:
in markdown, doing 

```
`__mode__=min`
```

is rendered as:

`__mode__=min`

Please generate the documentation to check that the modifications it renders are as you expect.

Reply:
Yes I wanted to make clear which argument should have which value